platform/ios/docs/guides/Runtime Styling.md

```diff
@@ -0,0 +1,56 @@
+# Dynamically Styling Maps
```

Comment:
The name of this file determines what appears in the navigation sidebar, so it and this heading should match.